documentation/governance.md

```diff
@@ -59,22 +57,26 @@ Distinct branches of the Standard will be maintained within Github for each vers
-* Major versions which make backwards-incompatible API changes; and
-* Minor versions which add functionality in a backwards-compatible manner.
+[Semantic Versioning](https://semver.org/) practices will be used to distinguish between:
+* MAJOR versions which make backwards-incompatible API changes; and
+* MINOR versions which add functionality in a backwards-compatible manner.
+* PATCH version which make backwards-compatible bug fixes.
 
-These are captured by a version number in the format MAJOR.MINOR
+These are captured by a version number in the format MAJOR.MINOR.PATCH.
 
 ### Revision process
-To release a new minor or major version upgrade will involve a number of stages outlined in the flowchart below, and described in more depth in the following sections.
+To release a new MINOR or MAJOR version upgrade will involve a number of stages outlined in the flowchart below, and described in more depth in the following sections.
 
 ![Revision process](../assets/upgrade_process_feb_2016.png)
 
+PATCH version upgrades are a smaller process, in recognition of the low impact that PATCH changes can have on the whole standard. A proposal for a PATCH version upgrade is made via the forum, with the Stewardship Comittee and technical team notified via email. If no objections are received within one week, the PATCH change will be considered approved.
+
 The revision process will follow these general principles:
 * **Publicity**: All stages of the revision process will be announced via the 360Giving online forum. This is the formal channel for notification during the process.
 * **Consensus**: The process should act in the interest of the data standard, with particular consideration given to what the changes will mean for current publishers. All processes should aim towards gaining community consensus for changes. In cases where consensus cannot be reached, the process will be escalated to the CEO of 360Giving and put to a final majority vote by the Stewardship Committee. The 360Giving technical team are responsible for generating key documentation during the process, but should always be guided by community consensus, submitting all decisions for public discussion.
 * **Appeal**: Any party may appeal against decisions made during the process by writing to the Standard Stewardship Committee via the 360Giving discussion forum. The Stewardship Committee has the authority to reject proposed revisions on the Standard in response to appeals.
 
+
 ### Proposals
 Changes to the Standard can be proposed by anyone at any point via the 360Giving discussion forum either as issues for discussion, or [pull requests]( https://help.github.com/articles/about-pull-requests/) with a clear description of the proposed change. Contributors are encouraged to raise discussions in order to seek consensus on proposed changes. Changes may be proposed as updated field definitions or codelist entries, or as new features to the Standard.
```

Comment:
Google doc states - MAJOR and MINOR changes...  so that needs to be added to line 81.

Reply:
I think that was actually in error. The following text actually applies to all levels of changes.